lang/cs/Org.Apache.REEF.Client/Common/HttpClientHelper.cs

```diff
@@ -90,7 +91,12 @@ internal static List<string> GetRmUri(string filePath)
             }
         }
 
-        internal static string GetAppId(string filePath)
+        /// <summary>
+        /// Return teh applicaiton id of the Job
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <returns></returns>
+        public string GetAppId(string filePath)
```

Comment:
May be not right now. Long term, once the clients get app id, they can use it to call Rest API.  I changed it back to internal for now until we need I.